test/unit/auth/user-import-builder.spec.ts

```diff
@@ -825,7 +835,7 @@ describe('UserImportBuilder', () => {
                 uid: 'enrollmentId2',
                 secret: 'SECRET',
                 displayName: 'Google Authenticator on personal phone',
-                factorId: 'totp',
+                factorId: 'factor',
```

Comment:
should be totp for a valid testcase? Maybe add another user with an invalid factor id and update the display name also?